src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class StorageDaoImplTest {
+    private static final StorageDao storageDao = new StorageDaoImpl();
+
+    @BeforeAll
+    public static void setupStorage() {
+        Storage.people.add(new User());
+        Storage.people.get(0).setLogin(null);
+        Storage.people.add(new User());
+        Storage.people.get(1).setLogin("admin");
+        Storage.people.add(null);
+    }
+
+    @Test
+    public void add_user_returnsSameUser() {
+        User user = new User();
```

Comment:
You can create a basic user with some fields and put it in a class variable. In each method, simply set the necessary fields for it, which will be tested in the method.